homeassistant/components/flux_led/__init__.py

```diff
@@ -94,7 +95,7 @@ async def async_update(self):
         bulb_list = await self.hass.async_add_executor_job(self._bulb_scan.scan)
         known_bulbs = self._config_entry.data[CONF_DEVICES]
         new_bulbs = {}
-        config_data = self._config_entry.data.copy()
+        config_data = copy.deepcopy(dict(self._config_entry.data.copy))
```

Comment:
I don't think we need to both copy the data and wrap it in a dict. Just the dict wrapping should be enough. Then you can deepcopy it.
```suggestion
        config_data = copy.deepcopy(dict(self._config_entry.data))
```